{
  "meta": {
    "desc": "Exception interfaces and implementations",
    "desc_long": null,
    "home": "https://pypi.python.org/pypi/zope.exceptions",
    "license": {
      "deprecated": false,
      "free": true,
      "fullName": "Zope Public License 2.0",
      "redistributable": true,
      "shortName": "zpl20",
      "spdxId": "ZPL-2.0",
      "url": "https://spdx.org/licenses/ZPL-2.0.html"
    },
    "maintainers": [
      {
        "email": "cillian.deroiste@gmail.com",
        "github": "cillianderoiste",
        "githubId": 643494,
        "name": "Cillian de R\u00f3iste"
      }
    ]
  },
  "versions": [
    {
      "commits": [
        {
          "first": "001b24bd25399d0aadc4d81e35563a13eb5a7a10",
          "last": "0031ccab55e72a1ca59128882f9f98de2a049226"
        }
      ],
      "name": "4.5"
    },
    {
      "commits": [
        {
          "first": "02ac13b563c082eaa51e8acdbe492a7961231836",
          "last": "00154ce2689a5affe0320c49e8b300cf9113447e"
        }
      ],
      "name": "4.4"
    }
  ]
}